src/components/ui/drawers/deletions/with-checkbox/DeleteConfirmationDrawer.tsx

```diff
@@ -98,6 +100,7 @@ export const DeleteConfirmationDrawer = ({
       });
       revalidatePage(path);
```

Comment:
When redirecting to `/sharehouses`, this `revalidatePage` function is no longer necessary as it doesn't need to re-fetch the data for the edit page.